### DISCUSSION TITLE
Docs Suggestion - Add Settings Dependencies in mastering-tabi-settings

### DISCUSSION THREAD

**Discussion by lylebrown** [welpo/tabi#364] [General]
I admit this may have been an issue because I am new to Zola, but it took me a while to realize that `show_previous_next_article_links` will not work without `sort_by` specified. It would be helpful for beginners if this detailed documentation page was very specific about what needs to be set for the feature/option to work.

**Comment by welpo** [comment]
Hey Lyle!

You're completely right.

I hadn't tested the feature without `sort_by`, so I wasn't aware of the limitation (though it makes sense).

I've updated the docs: https://github.com/welpo/tabi/commit/6219b9200cfd544b999b064fabfa0ca37d4815d8

Thank you for the heads up~

**Reply by lylebrown** [reply]
Awesome! I'm not sure if any other settings have similar prerequisites - if so, it would be good to add that info too.

**Reply by welpo** [reply]
I hope not! I try to make the documentation as clear as possible~
